src/styles/image.scss

```diff
@@ -0,0 +1,9 @@
+.images {
+  background-image: url(../images/imac.jpeg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 3px;
+}
```

Comment:
The class name `.images` suggests that it can be applied to multiple elements, but the styles within are very specific to a single image. Consider renaming the class to more accurately reflect its purpose, such as `.product-image` or `.card-image`, to improve readability and maintainability of the code.